### PR TITLE
TW-744 Add 'Savings' card on 'Earn' page

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

File: src/screens/savings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import React, { FC } from 'react';
+import { Text } from 'react-native';
+
+export const Savings: FC = () => <Text>TODO: Add some contents</Text>;
```
Comment (Boltman92) [inline src/screens/savings/index.tsx:4]: do we need this TODO for now?

Comment (keshan3262) [reply src/screens/savings/index.tsx:4]: I think we need it to prevent crashing on accidental card click

**Review by Boltman92 — APPROVED** [review]

File: src/screens/savings/index.tsx (hunk 1 above)
Comment (Boltman92) [reply src/screens/savings/index.tsx:4]: ok, no problem

**Review by Boltman92 — APPROVED** [review]

**Review by svjatoslavtt — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_757

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_757

**Pull request merged**